requirements.txt

```diff
@@ -4,6 +4,7 @@ qiskit-aer
 qiskit-nature[pyscf]
 qiskit-experiments
 qiskit-machine-learning
+qiskit-ibm-runtime>=0.19
```

Comment:
Is there a potential problem here that Runtime doesn't use Neko, so could merge breaking changes to the Neko actions unknowingly?

Reply:
During my searching for `backend_selection`, I also realised that `qiskit-algorithms` doesn't have a Neko job in CI, so could also break it without us knowing.

Reply:
> Do you think it could be worth dropping the backend_selection ability of the Aer plugin to get specific IBM-like devices? On a quick scan, I couldn't see any repos actually using the functionality.

> Is there a potential problem here that Runtime doesn't use Neko, so could merge breaking changes to the Neko actions unknowingly?

Yeah, we should add a CI job to qiskit-ibm-runtime if we go this path. We could potentially remove this functionality and just rely on users to make their own backend plugin to use the fake backends. I was using this locally to test things out when I was first getting neko setup, but I don't know how widely used this part of the interface was. The plugin interface supports it but each plugin [doesn't need to implement name selection](https://qiskit.org/ecosystem/neko/apiref/qiskit_neko.backend_plugin.BackendPlugin.get_backend.html#qiskit_neko.backend_plugin.BackendPlugin.get_backend).

> During my searching for backend_selection, I also realised that qiskit-algorithms doesn't have a Neko job in CI, so could also break it without us knowing.

Yeah we should add a job to qiskit-algorithms, we can ask @woodsp-ibm or @ElePT about it.

Reply:
Well we do have a jobs in Algorithms that run all the unit tests against Qiskit Main so at least from that side we can see and it did turn up an issue along the way to the final 1.0. They are not required for CI, all that is is done against stable release, but they are run in each PR and in the nightly CI run.

![image](https://github.com/Qiskit/qiskit-neko/assets/40241007/231b4ff6-7a6d-4349-b16b-717b10fa0359)

Reply:
Neko's a bit different to testing against Qiskit `main`, though - Algorithms running Neko in CI is / would be just as much a protection for all the other Neko users as for Algorithms itself.